### PR TITLE
Update marine-js public interface. Replace any with generic JSON types. Doing se\desalinization work insine marine-js

### PR DESCRIPTION
**Pull request by coder11**

### CODE REVIEW

**Review by ValeryAntopol — COMMENTED** [review]
perhaps greeting-record.wasm  update must be reverted

File: examples/failing/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Fluence Labs Limited
```
[the remark this answers is not in the document]
Comment (coder11) [reply examples/failing/src/main.rs]: fixed

File: marine-js/npm-package/src/__test__/test.spec.ts [hunk 2]
```diff
@@ -34,7 +35,7 @@ describe('Fluence app service tests', () => {
         await faas.init();
 
         // act
-        const res = JSON.parse(faas.call('greeting', '{"name": "test"}', undefined));
+        const res = faas.call('greeting', [{ name: 'test' }], undefined);
```
[the remark this answers is not in the document]
Comment (coder11) [reply marine-js/npm-package/src/__test__/test.spec.ts]: Oh, I see. So I've updated the example and added another test for the second type of arguments passing

**Review by shamsartem — APPROVED** [review]
I think is change is positive. A lot better to be more explicit on what is the actual interface

**Review by ValeryAntopol — APPROVED** [review]
utACK

**Pull request merged**